tests/core/config/test_data_node_config.py

```diff
@@ -46,14 +46,6 @@ def test_data_node_config_check():
         Config.configure_data_node("data_nodes", scope="bar")
         Config.check()
 
-    with pytest.raises(ConfigurationIssueError):
-        Config.configure_data_node("data_nodes", storage_type="sql")
-        Config.check()
-
-    with pytest.raises(ConfigurationIssueError):
-        Config.configure_data_node("data_nodes", storage_type="generic")
-        Config.check()
-
```

Comment:
Note: This 2 tests are removed because these methods will now return TypeError due to missing required properties